index.js

```diff
@@ -64,27 +76,8 @@ class PhotoViewManager {
     return scale;
   }
 
-  imageTouchHandler(e) {
```

Comment:
@shidhincr Now I do that on the event, on the Slider side, we capture and disable panning.